multilog/badger/qry.go

```diff
@@ -58,6 +59,23 @@ func (qry *query) Lte(s margaret.Seq) error {
 	return nil
 }
 
+func (qry *query) Reverse(yes bool) error {
+	qry.reverse = yes
```

Comment:
add check here that live is not set, and do a check in Live that reverse is not set

Reply:
done but needs to happen in the logs `Query()` since all the specs have to be applied first.